SU-HSDO/suhumsci
# Hsd8 531 refactor admin theme

# READY FOR REVIEW

# Summary
- Uninstall material admin theme and disable theme switcher module

# Needed By (Date)
- End of sprint 1/30

# Urgency
- high

# Steps to Test
1. checkout this branch
1. sync a sites `blt drupal:sync --site=[sitename]`
1. ensure admin theme looks like seven. 
1. validate on the appearance page, `Humsci Admin` is selected as the admin theme.
1. validate all content still appears on the page.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
